client/src/documentLinkProvider.ts

```diff
@@ -50,15 +99,12 @@ export class BitbakeDocumentLinkProvider implements vscode.DocumentLinkProvider
         link.range.end.line,
         link.range.end.character
       )
-      try {
-        const filename = link.value.split(';')[0]
 
-        const file = relatedFiles.find(file => file.path.endsWith(filename))
-        if (file !== undefined) {
-          documentLinks.push(new vscode.DocumentLink(range, vscode.Uri.parse(file.fsPath)))
-        }
-      } catch (err) {
-        logger.error(`An error occurred when finding files with pattern: ${link.value}. ${JSON.stringify(err)}`)
+      const filename = link.value.split(';')[0]
+
+      const file = foundFiles.find(file => file.path.endsWith(filename))
```

Comment:
not critical but this doesn't work if two files end with the same name. Ex:
`SRC_URI = "file://test.conf file://manual-test.conf"`

Reply:
one will point to the other

Reply:
Check 73899df1511e66953aee2bee0d03cca60c59099f